deploy-board/deploy_board/webapp/group_view.py

```diff
@@ -34,6 +35,7 @@
 
 
 DEFAULT_PAGE_SIZE = 50
+PHOBOS_URL = "https://phobos.pinadmin.com/phobos/"
```

Comment:
Add this url in internal teletraanconfigs and import it from deploy_board.settings 
See CMDB_URL 

Dont expose internal Urls